src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+.header {
+  max-width: 1200px;
+  left: 0;
+  top: 0;
+  margin: 0 auto;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: rgb(255, 255, 255);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+}
+
+.nav {
```

Comment:
Each block should be in a separate file.

Reply:
I thought that header should be  BEM block